ui/src/datasource/datasourceRest.ts

```diff
@@ -13,6 +13,12 @@ const http = axios.create({
   transformResponse: []
 })
 
+const httpSchema = axios.create({
```

Comment:
Pls extract the schema recommendation requests to another file like `schemaSuggestionRest.ts`